packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -420,10 +422,10 @@ class MediaQueryData {
         bottom: removeBottom ? 0.0 : null,
       ),
       viewPadding: viewPadding.copyWith(
-        left: math.max(0.0, viewPadding.left - padding.left),
-        top: math.max(0.0, viewPadding.top - padding.top),
-        right: math.max(0.0, viewPadding.right - padding.right),
-        bottom: math.max(0.0, viewPadding.bottom - padding.bottom),
+        left: removeLeft ? math.max(0.0, viewPadding.left - padding.left) : null,
```

Comment:
I think rather than null here it should just return the viewPadding.left and so on?

Reply:
Null in copywith just means that the value stays the same